docs/events.md

```diff
@@ -1,6 +1,8 @@
 @property {Object} can-component.prototype.events events
 @parent can-component.deprecated
 
+@deprecated Use [can-control] instance on the component custom tag instead.
```

Comment:
I’m not 100% sure what this means… can more info be provided? Providing more info inline would be great, or even a link to a CodePen would be useful.

Reply:
Done!